libbeat/scripts/generate_fields_docs.py

```diff
@@ -70,8 +70,21 @@ def fields_to_asciidoc(input, output, beat):
 
 
     docs = yaml.load(input)
+
+    # fields file is empty
+    if docs is None:
+        return
```

Comment:
I'm understanding correctly, these cases are errors. It seems like this check (and the others) should return or throw some kind of error so that the exit code is non-zero.